templates/default/rt/rtticketeditbox.html

```diff
@@ -124,7 +124,7 @@
 										{if !ConfigHelper::checkConfig('phpui.helpdesk_hide_disabled_users')
 											|| (ConfigHelper::checkConfig('phpui.helpdesk_hide_disabled_users') && $user.access)
 											|| $user.id == $ticket.owner}
-										<OPTION value="{$user.id}"{if !$user.access} class="blend"{/if}{if $user.id == $ticket.owner} selected{/if}>{$user.rname}</OPTION>
+										<OPTION value="{$user.id}"{if !$user.access} class="blend"{/if}{if $user.id == $ticket.owner} selected{/if}>{$user.name}</OPTION>
```

Comment:
Skąd nagle te zmiany?
Nie lubię jak ktoś pod szyldem jednej zmiany robi jednocześnie 5 innych zmian, bo potem ciężko się połapać, która zmiana za co odpowiadała?

Reply:
Wywaliłem z modules/rtticketadd.php bezpośrednie odwołanie do DB i zastąpiłem to funkcją która od dawna jest w systemie.